setup.cfg

```diff
@@ -0,0 +1,48 @@
+[metadata]
+name = pypinfo
+version = attr: pypinfo.__version__
+description = View PyPI download statistics with ease.
+long_description = file: README.rst
+author = Ofek Lev
+author_email = ofekmeister@gmail.com
+maintainer = Ofek Lev
+maintainer_email = ofekmeister@gmail.com
+url = https://github.com/ofek/pypinfo
+download_url = https://github.com/ofek/pypinfo
+license = MIT
+keywords =
+    bigquery
+    downloads
+    pypi
+    stats
+classifiers =
+    Development Status :: 4 - Beta
+    Intended Audience :: Developers
+    Intended Audience :: End Users/Desktop
+    License :: OSI Approved :: MIT License
+    Natural Language :: English
+    Operating System :: OS Independent
+    Programming Language :: Python
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.5
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3 :: Only
+    Programming Language :: Python :: Implementation :: CPython
+    Programming Language :: Python :: Implementation :: PyPy
+
+[options]
+python_requires = >=3.5
+install_requires =
+    appdirs
+    binary
+    click
+    google-cloud-bigquery >= 0.29.0
+    tinydb<4
```

Comment:
Let's have consistent spacing around the operator in these two lines

Reply:
Thanks for catching, I agree. Fixed in the latest revision.